bundles/Planning/Service/PlanningService.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of aakb/jira_economics.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Planning\Service;
+
+use App\Service\JiraService;
+
+class PlanningService extends JiraService
+{
+    /**
+     * Constructor.
+     */
+    public function __construct(
+        $jiraUrl,
+        $tokenStorage,
+        $customerKey,
+        $pemPath
+    ) {
+        parent::__construct($jiraUrl, $tokenStorage, $customerKey, $pemPath);
+    }
+
+    public function getBoard($boardId)
+    {
+        return $this->get('/rest/agile/1.0/board/'.$boardId);
+    }
+
+    /**
+     * Get all boards.
+     *
+     * @return array
+     */
+    public function getAllBoards()
+    {
+        $boards = [];
+
+        $start = 0;
+        while (true) {
+            $result = $this->get('/rest/agile/1.0/board?maxResults=50&startAt='.$start);
```

Comment:
The `get` function should be extended with a `array $query` parameter to make this easier to read (and use). An upcoming PR will add this.